src/Teams/Data/Repositories/MultipleAnswerQuestionRepository.cs

```diff
@@ -9,14 +9,31 @@ namespace Teams.Data.Repositories
 {
     public class MultipleAnswerQuestionRepository : IMultipleAnswerQuestionRepository
     {
-        ApplicationDbContext _db;
-        public MultipleAnswerQuestionRepository(ApplicationDbContext db)
+        IApplicationDbContext _db;
+        public MultipleAnswerQuestionRepository(IApplicationDbContext db)
         {
             _db = db;
         }
         public MultipleAnswerQuestion PickById(Guid Id)
         {
             return _db.MultipleAnswerQuestions.Include(a => a.Answers).Single(q => q.Id == Id);
         }
+
+        public void AddQuestion(MultipleAnswerQuestion question)
+        {
+            _db.MultipleAnswerQuestions.Add(question);
+        }
+
+        public void DeleteQuestionOptionsIn_DB(MultipleAnswerQuestion question)
```

Comment:
The most of naming convention guidelines suggested not to use underscores within method names. I would recommend the same.